Applications/ProductionManager/Contracts/Predict/CuttingPredictionRequest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Runtime.Serialization;
+
+namespace HomagConnect.ProductionManager.Contracts.Predict;
+
+/// <summary>
+/// Cutting Prediction Request
+/// </summary>
+public class CuttingPredictionRequest : IExtensibleDataObject
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest()
+    {
+        
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest(IEnumerable<CuttingPredictionPart>? productionEntities):this()
+    {
+        ProductionEntities = productionEntities;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest(IEnumerable<CuttingPredictionPart>? productionEntities, string? machine) :this(productionEntities)
```

Comment:
"parts" might be a better name for the variable.

Reply:
The name is still from the time, we wanted to have the "real and big" productionEntity in the request, so the model "xxxPredictionPart" was meant to be replaced.
Shall I rename it everywhere incl. docu to "xxxPredictionPart"?

Reply:
yes